ui/interfaces/data/login_data.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+
+class LoginData(BaseModel):
+    first_name: str
+    last_name: str
+    email: str
```

Comment:
```suggestion
    email: EmailStr
```

[https://docs.pydantic.dev/2.0/usage/types/string_types/](url)